tests/draft2019-09/optional/format/time.json

```diff
@@ -5,9 +5,74 @@
         "tests": [
             {
                 "description": "a valid time string",
+                "data": "08:30:06Z",
+                "valid": true
+            },
+            {
+                "description": "a valid time string with leap second",
+                "data": "23:59:60Z",
+                "valid": true
+            },
+            {
+                "description": "a valid time string with second fraction",
```

Comment:
I'll add another case with a different second fraction, since the spec uses `1 * DIGIT` to ensure that implementations do not restrict to a certain amount of second fractions. For example `23:20:50.52Z`.

Reply:
Added via 2edc74b